_data/simple-icons.json

```
@@ -4825,6 +4825,11 @@
             "hex": "0055FF",
             "source": "https://framer.com"
         },
+        {
+            "title": "Framework",
+            "hex": "000000",
+            "source": "https://frame.work/about"
```

```suggestion
            "source": "https://frame.work"
```
Link to front page is sufficient since there is no specific press kit page.